test/helpers/kubectl.go

```diff
@@ -1076,7 +1076,7 @@ func (kub *Kubectl) GetNodeNameByLabelContext(ctx context.Context, label string)
 
 // GetNodeIPByLabel returns the IP of the node with cilium.io/ci-node=label.
 // An error is returned if a node cannot be found.
-func (kub *Kubectl) GetNodeIPByLabel(label string, external bool) (string, error) {
+func (kub *Kubectl) GetNodeIPByLabel(label string, external bool, v4 bool) (string, error) {
```

Comment:
nit: maybe we can define and use an `IPFamily` enum to make a bit less cryptic the invocations of this function